src/styles/_catalog.scss

```diff
@@ -11,9 +13,18 @@
     align-items: center;
     width: 200px;
     height: 300px;
-    border: 1px solid #ccc;
+    border: 1px solid $border-color;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transform-origin: center;
+    transition-duration: 300ms;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated